e2e/wdio/platform/pages/button.po.ts

```diff
@@ -0,0 +1,41 @@
+import { BaseComponentPo } from './base-component.po';
+import { waitForPresent } from '../../driver/wdio';
+
+export class ButtonPo extends BaseComponentPo {
+    private url = '/menu-button';
+
+    typeButtons = 'fdp-button-types-example button';
+    sizeButtons = 'fdp-button-sizes-example button';
+    iconButtons = 'fdp-button-icons-example button';
+    stateButton = 'fdp-button-state-example button';
+    disableStateButtons = 'fdp-button-state-example button.is-disabled';
+    truncatedButton = 'fdp-button-truncate-example button';
+    
+    root = '#page-content';
+    static checkRtlSwitch() {
+        throw new Error('Method not implemented.');
```

Comment:
remove all these static methods. all methods are implemented - you just break the test - fails because it will always ready as method not implemented.

Reply:
remove = delete